lib/include/tree_sitter/api.h

```diff
@@ -655,62 +655,62 @@ bool ts_node_eq(TSNode, TSNode);
  * possible using the `TSNode` functions. It is a mutable object that is always
  * on a certain syntax node, and can be moved imperatively to different nodes.
  */
-TSTreeCursor ts_tree_cursor_new(TSNode);
+TSTreeCursor ts_tree_cursor_new(TSNode node);
 
 /**
  * Delete a tree cursor, freeing all of the memory that it used.
  */
-void ts_tree_cursor_delete(TSTreeCursor *);
+void ts_tree_cursor_delete(TSTreeCursor *_self);
```

Comment:
You can use normal self without underscore here and in other places in the api.h file. It used with underscore in implementations due to macro expansions.